examples/siro_sandbox/configs/social_nav_study.yaml

```diff
@@ -0,0 +1,4 @@
+# @package _global_
+
+defaults:
+  - fetch@_here_
```

Comment:
I don't think we should inherit from fetch config yaml. The underlying principle here is that someone should be free to tweak the fetch app state without breaking/affecting other app states. How about some kind of base sandbox config yaml that both fetch and social_nav_study inherit from?

Reply:
Yes, good catch!  Well, in this case the most secure decision is to have two independent config copies. 

In fact, both fetch and social_nav_study configs are not app state config but rather baselines configs overrides (I'd say finetuned versions of rearrange baselines config). So, how about having two independent copies? This is similar to what we do with app sates classes: copy instead of overrides.

Reply:
>are not app state config but rather baselines configs overrides
Agree! And we might say each is a baseline config override intended to be used with a particular app state (hence their names).

Reply:
I do still think a base sandbox config would make sense. I.e. a baseline config override intended to be used by all sandbox app use cases. And then fetch.yaml and social_nav_study.yaml can do a few additional overrides of that.